src/components/accessories/discharge/dischargeForm/DischargeForm.tsx

```diff
@@ -125,7 +125,9 @@ const DischargeForm: FC<DischargeProps> = ({
     enableReinitialize: true,
     onSubmit: (values) => {
       const formattedValues = formatAllFieldValues(fields, values);
-
+      formattedValues.disDate = moment(formattedValues.disDate)
+        .endOf("day")
+        .toISOString();
```

Comment:
I don't understand this formattedValues you added on the disDate, can you try to explain to me?

Reply:
I missed something there. Let me update !